pkg/rootless/rootless_linux.c

```diff
@@ -106,6 +106,10 @@ do_pause ()
   for (i = 0; sig[i]; i++)
     sigaction (sig[i], &act, NULL);
 
+  /* Attempt to execv catatonit to keep the pause process alive.  */
+  execl ("/usr/libexec/podman/catatonit", "catatonit", "-P", NULL);
```

Comment:
Some distributions (e.g., openSUSE) ship it in `/usr/bin/catatonit`.  Could we add another fallback for that?

Reply:
is it fine to assume it has support for `-P` even when distributed under `/usr/bin`?

Reply:
> is it fine to assume it has support for `-P` even when distributed under `/usr/bin`?

I think so since Podman now requires catatonit >= 0.17.0